cogs/misc.py

```diff
@@ -352,17 +352,17 @@ async def setafk(self, ctx, *, note: commands.clean_content = "I'm currently AFK
         if check is not None:
             await self.bot.db.execute("UPDATE afk SET message = $1 WHERE user_id = $2 AND guild_id = $3", note, ctx.author.id, ctx.guild.id)
             self.bot.afk[f"{str(ctx.guild.id)}, {str(ctx.author.id)}"]['note'] = note
-            await ctx.send(_("{0} **Changed your AFK state to -** {1}").format(
+            await ctx.send(_("{0} **Changed your AFK state to**: {1}").format(
```

Comment:
this should be before the asterisks

Reply:
if I understand how GitHub works, I fixed that